webapp/src/utils.js

```diff
@@ -32,3 +34,61 @@ export async function withWorking(errorHandler, action, ...params) {
     store.commit('working', false);
   }
 }
+
+function _digestError(error) {
+  if (error.response) {
+    // The request was made and the server responded with a status code
+    // that falls out of the range of 2xx
+    if (error.response.status < 500) {
+      // 3xx or 4xx
+      if (error.response.status === 401) {
+        if (sessionStorage.getItem('token')) {
+          setTimeout(() => clearToken());
+          return ['Session expired. Please login again.']
+        } else {
+          return ['You are not logged in.'];
+        }
+      } else if (error.response.status === 413) {
+        return ['Too much data. Try to reduce the length of your inputs.'];
+      } else if ('data' in error.response) {
+        let data = error.response.data;
+        if (typeof data === 'object') {
+          if ('detail' in data) {
+            return [data.detail];
+          } else if ('non_field_errors' in data) {
+            return data.non_field_errors;
+          } else {
+            return data;
+          }
+        } else {
+          console.log(`Received response of type ${typeof data}: ${data}`)
+          return [data];
+        }
+      } else {
+        return ["Server returned an empty response."];
+      }
+    } else {
+      // 5xx
+      if (error.response.status === 500) {
+        return ['Something went wrong at the server, but we currently do not know why. The support was already notified.'];
+      } else if (error.response.status === 503) {
+        return ['We are currently experiencing high load and thus unable to handle this request. Please try again later.'];
```

Comment:
This also happens during more invasive deployments, not only on high load. Perhaps fold into `else` case like:
`'Something went wrong at the server, but we currently do not know why.' Please try again later, and contact us if the problem persists for a longer time.`

Reply:
done